src/pages/contributors/jesse-glick.adoc

```diff
@@ -0,0 +1,107 @@
+= Jesse Glick
+:page-name: Jesse Glick
+:page-linkedin:
+:page-twitter: 
+:page-github: jglick
+:page-email:
+:page-image: avatar/jesse-glick.jpeg
+:page-pronouns:
+:page-location: Raleigh, North Carolina, USA
+:page-firstcommit: 2009
+:page-datepublished: 2024-11-05
+:page-featured: true
+:page-intro: Jesse Glick is a software engineer, currently residing in Raleigh, North Carolina and has been involved with the Jenkins project for over a decade. Prior to contributing to Jenkins, Jesse worked as an engineer at Sun Microsystems. After incorporating Jenkins to his work at Sun, albeit in a small way, there wasn't much more to do with the project. However, that all changed when Kohsuke reached out to Jesse to find out if he would be interested in working on Jenkins full time. Since joining CloudBees in 2012, Jesse has been an integral part of Pipeline development, and incorporating many functions that are now fundamental to current Pipeline operation. Jesse works constantly working on improving the Pipeline experience, and shares his insights on how to improve not only the Jenkins project, but the open-source workflow as a whole.
```

Comment:
```suggestion
:page-intro: Jesse Glick is a software engineer, currently residing in Raleigh, North Carolina and has been involved with the Jenkins project for over a decade. Prior to contributing to Jenkins, Jesse worked as an engineer at Sun Microsystems. After incorporating Jenkins to his work at Sun, albeit in a small way, there wasn't much more to do with the project. However, that all changed when Kohsuke reached out to Jesse to find out if he would be interested in working on Jenkins full time. Since joining CloudBees in 2012, Jesse has been an integral part of Pipeline development, and incorporating many functions that are now fundamental to current Pipeline operation. Jesse works constantly on improving the Pipeline experience, and shares his insights on how to improve not only the Jenkins project, but the open-source workflow as a whole.
```